directory/src/main/java/org/entcore/directory/services/impl/DefaultSchoolService.java

```diff
@@ -252,9 +252,18 @@ public void update(String structureId, JsonObject body, Handler<Either<String, J
 	}
 
 	@Override
-	public void getLevels(String structureId, UserInfos userInfos, Handler<Either<String, JsonArray>> results){
-		String filter =
-				"MATCH (s:Structure {id: {structureId}})<-[:DEPENDS]-(g:ProfileGroup)<-[:IN]-(u:User) ";
+	public void getLevels(String structureId, boolean inherit, UserInfos userInfos, Handler<Either<String, JsonArray>> results){
+		StringBuilder filter = new StringBuilder();
+		if (inherit) {
+			filter.append(
```

Comment:
Est-ce qu'on ne pouvait pas fusionner ça en une seule reqûete ?

`MATCH (parent:Structure {id: {structureId}})<-[:HAS_ATTACHMENT*0..]-(structure:Structure)<-[:DEPENDS]-(g:ProfileGroup)<-[:IN]-(u:User)`

Reply:
Si, fusionner les 2 MATCH renvoie le même résultat, en effet.